templates/workspaces/base/terraform/storage.tf

```diff
@@ -70,6 +75,7 @@ resource "azurerm_private_endpoint" "stgblobpe" {
 
   depends_on = [
     module.network,
+    azurerm_private_endpoint.stgfilepe
```

Comment:
Is this really required?
I say many times several PEs asked to be created at the same time and TF handles them sequentially all on its own.

Reply:
In my experience, yes. The storage account might throw errors that it is exclusively in use when multiple PEs are being created simultaneously.